src/sass/components/_feature-box.scss

```diff
@@ -2,6 +2,7 @@
   border: 1px solid #b6b6b6;
   text-align: center;
   margin-top: 40px;
+  height: calc(100% - 40px);
```

Comment:
Are you sure you can use calc method? 
@Varenthein What browser versions should we support?

Reply:
It works pretty well, and the support isn't that bad.